src/main/java/com/google/sps/servlets/Shared.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+
+public class Shared {
```

Comment:
Shared is a very uncommon name for a utilities class.  This is fine if you want to keep for now, but a more common way to do this would be to just have two files.

Put all the constants into a single file.
Put all the utility functions into another file.

Call the class something like Key, Constant or Property.

that way you would call it like:
Property.CORRELATOR

Reply:
done!